R/num.R

```diff
@@ -127,7 +127,7 @@ vec_arith.numeric.pillar_num <- vec_arith.pillar_num.default
 vec_math.pillar_num <- function(.fn, .x, ...) {
   "!!!!DEBUG vec_math(`v(.fn)`)"
 
-  out <- vec_math_base(.fn, vec_proxy(.x))
+  out <- vec_math_base(.fn, vec_proxy(.x), ...)
```

Comment:
Do other methods also need to forward the dots?

Reply:
Thank you for reviewing. As far as I can tell, I don't believe other methods need to forward the dots. `vec_math_base()` only takes function names as a character vector.

``` r
library(sloop)
library(vctrs)
library(pillar)

vec_math_base("sum", c(1:3, NA), na.rm = TRUE)
#> [1] 6

s3_dispatch(vec_math_base("sum", c(1:3, NA), na.rm = TRUE))
#>    vec_math_base.character
#>    vec_math_base.default

s3_methods_class("pillar_num")
#> # A tibble: 9 × 4
#>   generic            class      visible source             
#>   <chr>              <chr>      <lgl>   <chr>              
#> 1 format             pillar_num FALSE   registered S3method
#> 2 vec_arith.numeric  pillar_num FALSE   registered S3method
#> 3 vec_arith          pillar_num FALSE   registered S3method
#> 4 vec_cast.double    pillar_num FALSE   registered S3method
#> 5 vec_math           pillar_num FALSE   registered S3method
#> 6 vec_ptype_abbr     pillar_num FALSE   registered S3method
#> 7 vec_ptype_full     pillar_num FALSE   registered S3method
#> 8 vec_ptype2.double  pillar_num FALSE   registered S3method
#> 9 vec_ptype2.integer pillar_num FALSE   registered S3method
```

<sup>Created on 2024-06-20 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>